tests/hooks/use-suggestions.spec.ts

```diff
@@ -121,8 +121,13 @@ describe('use-suggestions', () => {
     })
 
     it('suggestions should have the generic operators', () => {
+        const genericOperators = ['=', '!=', 'IN', 'NOT-IN']
+
         findSuggestions('Level ')
-        expect(suggestions.value).toEqual(['=', '!=', 'IN', 'NOT-IN'])
+        expect(suggestions.value).toEqual(genericOperators)
+
+        findSuggestions('Level =')
+        expect(suggestions.value).toEqual(genericOperators)
```

Comment:
why should the suggestion here be the generic operators ?

Reply:
@guyDataloop request (if we press backspace after applying an operator from suggestions, we should see a list of operators)

Reply:
@makc-dataloop would this still add a space where needed?

Reply:
@fadiDL this should not change the behavior for `Level = ` since we have a test for that and it passes